### ISSUE TITLE
HJKL --> JKL;

### ISSUE THREAD

**Issue by t-h2o** [jomccr/firefox-hjkl#2]
Hello,
Can you please add setting for change keys?
I prefer "JKL;" than "HJKL"

**Comment by jomccr** [comment]
Yeah I'll look into this but it might take a while, I'm not too sure how to go about it yet but I will try!

**Comment by jomccr** [comment]
I know I'm 3.5 years late, but this was actually just fixed in version 1.1 lol, give it a shot and let me know what you think! 

https://addons.mozilla.org/firefox/addon/vim-hjkl/

**Comment by t-h2o** [comment]
Thanks @jomccr.

After 3.5 years, I changed my preferences and I selected "HJKL" instead of "JKL;".

I got used to the default setting, it is clearly better than changing every application.

Thanks for your reply.

> Better late than never.
